README.md

```diff
@@ -29,7 +29,11 @@ git clone https://github.com/o3de/o3de.git
 ```
 
 ## Building the Engine
-### Build Requirements and redistributables
+
+### Build requirements and redistributables
+
+For the latest details and system requirements, refer to [System Requirements](https://o3de.org/docs/welcome-guide/requirements/) in the documentation.
+
 #### Windows
 
 *   Visual Studio 2019 16.9.2 minimum (All versions supported, including Community): [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/)
```

Comment:
Hmmm, these requirements don't match what we have in the welcome guide https://o3de.org/docs/welcome-guide/requirements/ regarding the VS version (it's a range) and CMake is actually 3.20.5

Reply:
Updated and clarified.